src/page/StoreRegistration/component/TimePicker/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState, useRef, useEffect } from 'react';
+import styles from './TimePicker.module.scss';
+
+const hours: number[] = Array.from({ length: 24 }, (_, i) => i);
+const minutes: number[] = Array.from({ length: 12 }, (_, i) => i * 5);
+
+export default function OperatingHour() {
+  const dropMenuRef = useRef<HTMLDivElement | null>(null);
+  const [isOpen, setIsOpen] = useState(false);
+  const [time, setTime] = useState({
+    hour: '00',
+    minute: '00',
+  });
+
+  useEffect(() => {
+    const handleOutsideClose = (e: { target: any }) => {
+      if (isOpen && (!dropMenuRef.current?.contains(e.target))) setIsOpen(false);
+    };
+    document.addEventListener('click', handleOutsideClose);
+
+    return () => document.removeEventListener('click', handleOutsideClose);
+  }, [isOpen]);
+
+  function toggleModal() {
+    setIsOpen(!isOpen);
+  }
+
+  function handleTimeChange(e: any) {
```

Comment:
바로 위 리뷰를 달고 확인해봤는데 any라서 문제가 없었나보네요. any는 최~~대한 쓰지 말아주세요오
```suggestion
  function handleTimeChange(e: ChangeEvent<HTMLInputElement>) {
```

그런데 button의 click이벤트에 담은 핸들러인데 어떻게 `e.target.value` 를 갖는거죠?

Reply:
button의 click이벤트에 `e.target.value`가 있는 줄 알았습니다.. 해당 부분은 MouseEvent로 수정하고 `e.currentTarget.value`로 수정했습니다. 0e64dffd02dbde058fffdbcc6830ecb2909665f0, 1e8ab8d9f58d8b4bfbfa0d5f07530f63994cbd2d